app_demo.rb

```diff
@@ -0,0 +1,103 @@
+require './lib/order'
+require './lib/submitorder'
+
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1, :total, :to_mobile, :from_mobile, :message_body, :message, :account_sid, :auth_token, :submitorder1, :quantity
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+  def add_dish
+    while true
+      puts 'Add dishes name and then price (press return twice to end order )'
+      @name = gets.chomp
+      break if name == ''
+      @price = gets.chomp
+      p menu1.add_dish name, price.to_f
+    end
+  end
+
+  def submit_order
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for submit_order is too high. [28.18/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [16/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)